AdminOpsGuide/dbrepl.rst

```diff
@@ -1191,141 +1193,312 @@ Here is the code:
 Setting up an A -> B Replication Configuration with Empty Databases
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-On A:
+**On A**:
+
+* Source the ydbenv environment file.
+
+.. parsed-literal::
+   source ./ydbenv A r122
+
+* Create the database and set up replication.
+
+.. parsed-literal::
+   $./db_create
+   %GDE-I-LOADGD, Loading Global Directory file
+        /home/A/yottadb.gld
+   %GDE-I-VERIFY, Verification OK
+
+   %GDE-I-EXECOM, Executing command file /home/A/gdemsr
+   %GDE-I-VERIFY, Verification OK
+
+   %GDE-I-GDUPDATE, Updating Global Directory file
+        /home/A/yottadb.gld
+   %YDB-I-DBFILECREATED, Database file /home/A/yottadb.dat created
+
+.. parsed-literal::
+   $./repl_setup
+   %YDB-I-JNLCREATE, Journal file /home/A/yottadb.mjl created for region DEFAULT with BEFORE_IMAGES
+   %YDB-I-JNLSTATE, Journaling state for region DEFAULT is now ON
+   %YDB-I-REPLSTATE, Replication state for region DEFAULT is now ON
 
-* Turn on replication.
-* Create the replication instance file.
 * Start the Source Server.
 
+.. parsed-literal::
+   ./originating_start A B 4001
+   Wed Jul 18 12:45:46 2018 : Initiating START of source server for secondary instance [B]
+   Wed Jul 18 12:45:46 2018 : Initiating CHECKHEALTH operation on source server pid [18498] for secondary instance name [B]
+   PID 18498 Source server is alive in ACTIVE mode
+   Wed Jul 18 12:45:46 2018 : /usr/local/lib/yottadb/r122/mupip replicate -source -start -instsecondary=B -secondary=localhost:4001 -buffsize=1048576 -log=/home/A/A_B.log
+   Wed Jul 18 12:45:46 2018 : %YDB-I-REPLINFO, GTM Replication Source Server with Pid [18498] started for Secondary Instance [B]
+   Wed Jul 18 12:45:46 2018 : Created jnlpool with shmid = [1353941014] and semid = [1447755780]
+   Wed Jul 18 12:45:46 2018 : %YDB-I-REPLINFO, GTM Replication Source Server now in ACTIVE mode using port 4001
+   Wed Jul 18 12:45:46 2018 : Connect hard tries count = 5, Connect hard tries period = 500
+   Wed Jul 18 12:45:46 2018 : 1 hard connection attempt failed : Connection refused
+
+Once B is brought up as replicating, check the log created. For example,
```

Comment:
It would be better to include this section (with the output) after the step where we illustrate bringing B up.

Reply:
But this is run on A, so do you want me to create another whole subsection of A again? I'm trying to make it clear by just having everything that happens on A together, and then everything that happens on B. Checking the log is not mandatory, it's more of an add-on, showing that you can look at it on A later.

Reply:
I think it would be better for the user if the flow of reading matches the flow of steps they need to run. And at each step indicate whether it is done on A or B. If having a subsection each time a switch of instance happens makes it better, I think that is fine.

Reply:
Changed the location of the section to match the flow.